packages/adp-tooling/package.json

```diff
@@ -28,6 +30,7 @@
     "files": [
         "LICENSE",
         "templates",
+        "rta",
```

Comment:
Shouldn't this be `rta.d.ts`?

Reply:
That is outdated, it shouldn't be there at all.

Reply:
8287c49

Reply:
Great